### ISSUE TITLE
Proposal to simplify the `getCalendarYear` method output

### ISSUE THREAD

**Issue by 9mbs** [9mbs/calendar-widgets#81]
This proposal suggests simplifying the output of the `getCalendarYear` method by replacing the current output with a more concise one.

Currently, the `getCalendarYear` method returns an object representing a calendar year with the number of days and a list of days for each month. The output includes nested objects with the month name as the key and an object containing the count and collection as the value. This output structure can be complex and challenging to work with for users who want to extract the data.

To simplify the output, this proposal suggests changing the output to a flat object that includes the month name as the key and the list of days as the value. This output structure will be more straightforward and easier to work with for users who want to extract the data.

**Comment by Ngoakor12** [comment]
This would definitely simplify everything. I struggled to think of a place where the count would be useful. Even if there were an instance where you would use it, you could just use the length of the month array values :slightly_smiling_face:

**Comment by 9mbs** [comment]
agreed! I was thinking the same thing. I'll make this as `help wanted`. It's yours if you want it @Ngoakor12, no pressure! 🙂
